index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    
+    <header>
+        <div class="logo">Vishal</div>
+       
+        <div class="burger">
+            <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
+        </div>
+    </header>
+        <section class="content">
```

Comment:
- [ ] Your design looks good 👍🏽. I think it will be better to reduce the padding-top between the toolbar and the headline section as the spacing is quite too much. Kindly check the style property on figma to better get the accurate spacing 😄 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/228682708-43e43a08-5fbb-4682-9816-c3419353aab8.png)

figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/228682838-d85ad46c-a4ed-4715-9c9d-4e2a859e5cca.png)